lambda/tasks/views.py

```diff
@@ -55,8 +55,10 @@ class UnassignedTaskListView(mixins.LoginRequiredMixin, generic.ListView):
     def get_queryset(self):
         q = Task.objects.none()
 
-        for project in Project.objects.filter(team__leader=self.request.user):
-            q = q | project.task_set.filter(assigned_to=None)
+        for project in (Project.objects.filter(team__leader=self.request.user) | Project.objects.filter(team__members__in=[self.request.user])):
+            q = q | project.task_set.filter(assigned_to=None).filter(
+                completed=False
```

Comment:
Is it necessary to check if a task is completed and unassigned? Does a task not need to be assigned before being completed?